Fix Minor Typo on API Concepts Page (#33088)
kubernetes/website: content/en/docs/reference/using-api/api-concepts.md

GET).

For some resources, the API includes additional subresources that allow

> fine grained authorization (such as separate viewing details for a Pod from

Reviewer: This still doesn't sound right to me, therefore re-requesting changes.

```suggestion
fine grained authorization (such as separate views for Pod details and
```

> fine grained authorization (such as separating viewing details for a Pod from

Reviewer: Slight modification because the sentence doesn't read correctly with your change. It didn't read correctly before, either.
```suggestion
fine grained authorization (such as separate viewing details for a Pod from
```

> retrieving its logs), and can accept and serve those resources in different

Reviewer: ```suggestion
log retrievals), and can accept and serve those resources in different
```
